algorithms/CPlusPlus/Arrays/segregate0and1.cpp

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
The file name should be `segregate-0-and-1.cpp`

Reply:
Hi @ming-tsai 
Renamed the file as mentioned as well updated the readme file.